yash-datalake/nested/storageAccount.json

```diff
@@ -31,7 +31,7 @@
     {
       "type": "Microsoft.Storage/storageAccounts",
       "name": "[parameters('storageAccountName')]",
-      "apiVersion": "2019-06-01",
+      "apiVersion": "2018-11-01",

```

Comment:
why the rollback?

Reply:
This API version was creating problem while running spark job in data factory. Tested all possible API version came out to the conclusion that Storage account and azure functions need to have similar API versions.
That's I roll backed it to the version where both are same

i.e 2018-11-01